Basic/DynamicAddressablesNetworkPrefabs/README.md

```diff
@@ -1,27 +1,82 @@
-# Dynamic Prefabs
+# Dynamic Addressables Network Prefabs Sample
 
-This sample showcases the available use-cases for the dynamic prefab system, which allows us to add new spawnable prefabs in runtime.
+![UnityVersion](https://img.shields.io/badge/Unity%20Version:-2021.3%20LTS-57b9d3.svg?logo=unity&color=2196F3)
+![NetcodeVersion](https://img.shields.io/badge/Netcode%20Version:-1.2.0-57b9d3.svg?logo=unity&color=2196F3)
+[![LatestRelease](https://img.shields.io/badge/Latest%20%20Github%20Release:-v1.2.0-57b9d3.svg?logo=github&color=brightgreen)](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/releases/tag/v1.2.0)
+<br><br>
 
-## Sample feature
+The Dynamic Prefabs Sample showcases the available use-cases for the dynamic prefab system, which allows us to add new spawnable prefabs at runtime. This sample uses Addressables to load the dynamic prefab, however any GameObject with a NetworkObject component can be used, regardless of its source. This sample also uses in-game UI (created using UI Toolkit) to interface with the dynamic prefabs system with configurable options like artificial latency and network spawn timeout for easy testing.
+<br><br>
 
-Dynamic prefabs feature allows the developer to add a new prefab to the network prefab list in runtime.
-For simplicity, this sample uses Addressables to load the dynamic prefab, however any GameObject with a NetworkObject component can be used, regardless of it's source (think User Generated Content (UGC)).
+# Sample Overview
 
-There are several limitations to this API:
-- If you have NetworkConfig.ForceSamePrefabs enabled, you can only modify your prefab lists before starting
+In this sample, learn more about:
+- The dynamic prefabs system
+- Loading prefabs with addressables
+<br><br>
+
+## Exploring the Sample
+
+Each scene in the project showcases a different, isolated feature of the API, allowing for easy extraction into other projects. We suggest exploring them in order to get a good understanding of the flow of dynamically loading and spawning network prefabs. The use-cases available in this sample are based around the current known limitations.
+<br><br>
+
+### Each Scene:
+
+- Scene 00_Preloading Dynamic Prefabs
+  - This is the simplest case of a dynamic prefab - we instruct all game instances to load a network prefab (it can be just one, it could also be a set of network prefabs) and inject them to NetworkManager's NetworkPrefabs list before starting the server.
```

Comment:
```suggestion
  - This is the simplest case of a dynamic prefab - we instruct all game instances to load a network prefab (it can be just one, it could also be a set of network prefabs) and inject them to NetworkManager's NetworkPrefabs list before starting the server.
  - This is the less intrusive option for your development, as you don't have any additional spawning and addressable management to do later in your game. 
```